realm-transformer/src/main/groovy/io/realm/transformer/RealmTransformer.groovy

```diff
@@ -44,10 +42,42 @@ import static com.android.build.api.transform.QualifiedContent.*
 class RealmTransformer extends Transform {
 
     private Logger logger = LoggerFactory.getLogger('realm-logger')
-    private Project project
+    TransformerConfigProvider provider
 
-    public RealmTransformer(Project project) {
-        this.project = project
+    public RealmTransformer(TransformerConfigProvider provider) {
+        this.provider = provider
+    }
+
+    // constructor for OkBuck
+    public RealmTransformer(File configFile) {
+        if (!configFile.exists()) {
+            throw new FileNotFoundException(configFile.absolutePath)
```

Comment:
You can expect this to always exist. Transform-cli does that check if a config file is specified: https://github.com/uber/okbuck/commit/ebcea6f0950d37536e18e0bfe7ae6eb39ebf4064#diff-10fc19e7dd1e6127839942575e0a0ba1R59